### ISSUE TITLE
Model cannot download from Google Drive

### ISSUE THREAD

**Issue by Dys2X** [ardha27/AI-Song-Cover-RVC#42]
Hello there, 

it's been a long time since i did AI Covers. 
I had to reload my Model voice from Google drive and it seemed to work, because it said "done". 
But when i go to Inference after and refresh the site. "Model Voice" is empty. 
Did i do anything wrong? 

Looking forward for your help

**Comment by MinatoIsuki** [comment]
I don't use GitHub as often now, but, have you try to upload it on HuggingFace?

**Comment by Dys2X** [comment]
> I don't use GitHub as often now, but, have you try to upload it on HuggingFace?

Yes, it seems that it doesn't work there either.
![image](https://github.com/ardha27/AI-Song-Cover-RVC/assets/146621367/9ec2ef30-1230-425a-950f-440f02b9adf7)

**Comment by MinatoIsuki** [comment]
> Yes, it seems that it doesn't work there either. ![image](https://private-user-images.githubusercontent.com/146621367/326728780-9ec2ef30-1230-425a-950f-440f02b9adf7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0ODM0MTEsIm5iZiI6MTcxNDQ4MzExMSwicGF0aCI6Ii8xNDY2MjEzNjcvMzI2NzI4NzgwLTllYzJlZjMwLTEyMzAtNDI1YS05NTBmLTQ0MGYwMmI5YWRmNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQzMFQxMzE4MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZjM0NjJkMTA5YTlkYjM3MWNiNDhhOTI2ODFiMmQ5NDY4ZjViMzNjMjEyM2I2ZjIxOTExNDg5ODVhNDFlMTQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.n2aDhKFfyv_Wm-57Nz5PMPf51wo6rxbpedh-cHLvLZM)

No, I mean uploading is by make a model and upload your zip file to there.
![image](https://github.com/ardha27/AI-Song-Cover-RVC/assets/102457568/e3e4f3d6-a050-497e-8b15-714a9acaeb6e)

**Comment by Dys2X** [comment]
> > Yes, it seems that it doesn't work there either. ![image](https://private-user-images.githubusercontent.com/146621367/326728780-9ec2ef30-1230-425a-950f-440f02b9adf7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0ODM0MTEsIm5iZiI6MTcxNDQ4MzExMSwicGF0aCI6Ii8xNDY2MjEzNjcvMzI2NzI4NzgwLTllYzJlZjMwLTEyMzAtNDI1YS05NTBmLTQ0MGYwMmI5YWRmNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQzMFQxMzE4MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZjM0NjJkMTA5YTlkYjM3MWNiNDhhOTI2ODFiMmQ5NDY4ZjViMzNjMjEyM2I2ZjIxOTExNDg5ODVhNDFlMTQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.n2aDhKFfyv_Wm-57Nz5PMPf51wo6rxbpedh-cHLvLZM)
> 
> No, I mean uploading is by make a model and upload your zip file to there. ![image](https://private-user-images.githubusercontent.com/102457568/326821728-e3e4f3d6-a050-497e-8b15-714a9acaeb6e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0ODQxMTQsIm5iZiI6MTcxNDQ4MzgxNCwicGF0aCI6Ii8xMDI0NTc1NjgvMzI2ODIxNzI4LWUzZTRmM2Q2LWEwNTAtNDk3ZS04YjE1LTcxNGE5YWNhZWI2ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQzMFQxMzMwMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZDQwOGFmYWZjZDhhMjRlYjg5NTdmOTAyYzc4MGY1ODk2NmZhMDJmMjU2OGQxOTFkMDYyZDM1ZmVkMDg0YjA5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.UveRj2V4okZ6jr90ycKiQFB1yWM99k3AC8aHHDV6cKg)

Wait, what? Where can i do this? I didn't see this option.

**Comment by MinatoIsuki** [comment]
> Wait, what? Where can i do this? I didn't see this option.

Have you make an account yet? The menu should be right at the Home screen (https://huggingface.co)

**Comment by Dys2X** [comment]
Yes, got it, thanks! But what about the license? Which one should i use and what is it for?

**Comment by MinatoIsuki** [comment]
> Yes, got it, thanks! But what about the license? Which one should i use and what is it for?

You can leave it unlicensed, since it really doesn't matter much.
